src/applications/vaos/tests/new-appointment/components/PreferredDatePage.unit.spec.jsx

```diff
@@ -1,133 +1,145 @@
 import React from 'react';
 import { expect } from 'chai';
-import sinon from 'sinon';
-import { mount } from 'enzyme';
 import moment from 'moment';
 
-import { PreferredDatePage } from '../../../new-appointment/components/PreferredDatePage';
+import PreferredDatePage from '../../../new-appointment/components/PreferredDatePage';
+import { createTestStore, renderWithStoreAndRouter } from '../../mocks/setup';
+import { mockFetch, resetFetch } from 'platform/testing/unit/helpers';
+import { fireEvent } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingVSPAppointmentNew: false,
+    vaOnlineSchedulingDirect: true,
+    vaOnlineSchedulingCommunityCare: true,
+  },
+  user: {
+    profile: {
+      facilities: [
+        { facilityId: '983', isCerner: false },
+        { facilityId: '984', isCerner: false },
+      ],
+    },
+  },
+};
+
+describe('VAOS integration: preferred date page with a single-site user', () => {
+  beforeEach(() => mockFetch());
+  afterEach(() => resetFetch());
 
-describe('VAOS <PreferredDatePage>', () => {
   it('should render', () => {
-    const openFormPage = sinon.spy();
-    const updateFormData = sinon.spy();
-
-    const form = mount(
-      <PreferredDatePage
-        openFormPage={openFormPage}
-        updateFormData={updateFormData}
-        data={{}}
-      />,
-    );
-
-    expect(form.find('DateWidget').length).to.equal(1);
-    form.unmount();
-  });
+    const store = createTestStore(initialState);
+    const screen = renderWithStoreAndRouter(<PreferredDatePage />, {
+      store,
+    });
 
-  it('should not submit empty form', () => {
-    const openFormPage = sinon.spy();
-    const history = {
-      push: sinon.spy(),
-    };
-
-    const form = mount(
-      <PreferredDatePage
-        openFormPage={openFormPage}
-        history={history}
-        data={{}}
-      />,
+    expect(screen.baseElement).to.contain.text(
+      'What is the earliest date you’d like to be seen?',
     );
+    expect(screen.baseElement).to.contain.text(
```

Comment:
I think we'd also want to include an expect statement that verified that the date widget was on the page, probably by checking that there are two comboboxes and a textbox using `getByRole`.

Reply:
Good call, adding checks now.